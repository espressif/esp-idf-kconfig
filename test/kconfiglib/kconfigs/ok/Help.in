# Test if help text is parsed correctly
mainmenu "Main menu"

    config HELP
        bool
        default y
        help
            This is a long help with and example of a command which should be parsed as is:

            espfuture.py --chip esp64 burn_linux

            The output should look like:

            Distro: {Fedora 42}

                * RAM: 64 MB
                * CPU: Triple core HTP
                ...
            Done.

            And this is the end.

    config JUST_CHECKING
        bool "Prompt"
        default y
        help
            Just checking if the parsing of previous help text really stopped.

    comment "If you see this in the output, the parsing (probably) succeeded."
