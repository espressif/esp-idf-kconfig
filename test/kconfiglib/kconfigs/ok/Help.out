#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#
# default:
CONFIG_HELP=y
# default:
CONFIG_JUST_CHECKING=y

#
# If you see this in the output, the parsing (probably) succeeded.
#
